docs/superpowers/specs/2026-05-18-pattern-art-design.md

```
@@ -0,0 +1,429 @@
+
+```tsx
+<Pattern seed="Family Networks 2024" />                   // variant derived from seed
+<Pattern seed="Family Networks 2024" variant="dots" />    // explicit variant
+<DotsPattern seed="Family Networks 2024" />               // direct variant access
+```
+
+Each renders an inline `<svg>` sized to the parent via CSS, containing a solid background rect and a pattern composed of shapes drawn in three accent colors from the Network Canvas palette. All values — palette, geometry, variant (when omitted) — are derived from one PRNG seeded by the input string.
+
+## File layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)